qemu/tests/qemu_guest_agent.py

```diff
@@ -1721,6 +1721,101 @@ def gagent_check_path_fsfreeze_hook(self, test, params, env):
         else:
             test.fail("The default path of fsfreeze-hook doesn't match with expectation.")
 
+    @error_context.context_aware
```

Comment:
please put all your command into config.

Reply:
update last week, sorry to forget update comment here.